src/Redux/Missions/missions.js

```diff
@@ -11,7 +12,6 @@ const fetchMissions = createAsyncThunk(
   FETCHED_MISSIONS,
   async () => {
     const { data } = await axios.get(missionsAPI);
-    // eslint-disable-next-line camelcase
```

Comment:
You also repeat it here eslint disable, pleas  take it them in to consideration!

Reply:
@bernabasy  thanks for the suggestions